### PR TITLE
docs: Add EBS CSI Driver permissions to 2.9

### LINKED ISSUE #1453: Missing EKS permissions for EBS CSI Driver in 2.9
## Summary

The [docs for 2.9 ](https://ranchermanager.docs.rancher.com/v2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks#minimum-eks-permissions) are missing the EBS CSI Driver permissions for EKS. The missing permissions [are however present in latest](https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks#minimum-eks-permissions).

### PR DESCRIPTION
**Pull request by yiannistri**
Fixes #1453 

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, remember to add a "MERGE ON RELEASE" label and set the proper milestone.

## Description

Added missing permissions to the relevant section for EKS.

### CODE REVIEW

File: versioned_docs/version-2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md [hunk 1]
```diff
@@ -314,6 +314,43 @@ These are permissions that are needed by Rancher to create a Virtual Private Clo
 }
 ```
 
+### EBS CSI Driver addon Permissions
+
+Permissions required for Rancher to install the Amazon EBS CSI Driver add-on.
```
Comment (martyav) [inline versioned_docs/version-2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md:319]:
```suggestion
The following are the required permissions for installing the Amazon EBS CSI Driver add-on.
```

**Comment by martyav** [comment]
We will also need to update the corresponding file in /docs

**Comment by yiannistri** [comment]
Thanks for the review @martyav  I have added a few more lines to keep 2.9 consistent with latest, as per https://github.com/rancher/rancher-docs/pull/704

**Comment by yiannistri** [comment]
Ping @martyav for another review :pray:

**Review by martyav — DISMISSED** [review]
~Really minor suggestion for punctuation~

File: versioned_docs/version-2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md [hunk 2]
```diff
@@ -314,6 +318,43 @@ These are permissions that are needed by Rancher to create a Virtual Private Clo
 }
 ```
 
+### EBS CSI Driver addon Permissions
+
+The following are the required permissions for installing the Amazon EBS CSI Driver add-on.
```
Comment (martyav) [inline versioned_docs/version-2.9/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/set-up-clusters-from-hosted-kubernetes-providers/eks.md:323]:
```suggestion
The following are the required permissions for installing the Amazon EBS CSI Driver add-on:
```

**Review by martyav — DISMISSED** [review]
Really minor suggestion, let's get this made

**Review by martyav — APPROVED** [review]

**Pull request merged**